config/initializers/new_framework_defaults_7_2.rb

```diff
@@ -0,0 +1,74 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
(We can work through these changes in subsequent PR(s), and eventually delete this file and update from `config.load_defaults(7.1)` to `config.load_defaults(7.2)`.)

Reply:
Ticket: https://davidrunger.atlassian.net/browse/DEV-88